README.md

```diff
@@ -40,32 +39,51 @@ runner pod and engine monitor service), which are created & managed by it in ord
 The ChaosEngine is the core schema that defines the chaos workflow for a given application. Currently, it defines the following:
 
 - Application Data (namespace, labels, kind)
-- List of Chaos Experiments to be executed
-- Attributes of the experiments, such as, rank/priority 
-- Execution Schedule for the batch run of the experiments
+- ServiceAccount used for execution of the experiment
+- Chaos Experiment to be executed on the application
+- Attributes of the experiments (overrides defaults specified in the experiment CRs)
+- Chaos resource cleanup Option 
```

Comment:
Can we also add some details about the auxiliaryAppInfo and anotationCheck field?

Reply:
considering we are in master, it should be fine. But taking into account users may copy from here, reverted to previous schema and added note to say (as of 1.0.0)